### ISSUE TITLE
Provide scaler for Riak TS

### ISSUE THREAD

**Issue by kozdincer** [kedacore/keda#1790]
### Proposal

Provide scaler for Riak TS to scale on your metrics:

### Scaler Source

https://riak.com/products/riak-ts/index.html

### Scaling Mechanics

It specifically for IoT and Time Series data. It ingests, transforms, stores, and analyzes massive amounts of time series data.

### Authentication Source

TBD

### Anything else?

I think riak TS and riak KV should be written as separate scalers (as in redis-lists and redis streams)

**Comment by coderanger** [comment]
Basho, the company behind Riak, has been defunct since 2017. While it does live on a bit via open source I think this is probably too niche for core inclusion and should be a third party plugin. (Sadly)

**Comment by kozdincer** [comment]
New version for riak kv was released yesterday. But I understand your concerns. maybe 3rd part scaler is the best. I understand.
Thank you.

**Comment by coderanger** [comment]
Yes Riak KV and maybe CS have seen some continued life, I don’t think TS has? I can’t find much info on it post Basho.

**Comment by kozdincer** [comment]
Yes you are right. I don't even know how suitable the metrics to be taken from kv vs cs for using in keda.
Thanks for discussion.

**Comment by tomkerkhove** [comment]
So what I'd propose is to look at an external scaler then and see what the adoption is of it?

We're happy to have a reference to it on https://github.com/kedacore/external-scalers!

**Comment by tomkerkhove** [comment]
@kozdincer Just out of curiosity - Did you take it further with this scaler @kozdincer ?

**Comment by kozdincer** [comment]
@tomkerkhove No unfortunately

**Comment by tomkerkhove** [comment]
No problem at all!
